README.md

```diff
@@ -106,6 +106,13 @@ tagging = client.tags(tag_id).taggings.get(tagging_id)
 
 # Delete a Tagging
 client.tags(tag_id).taggings.delete(tagging_id)
+
+# look up events in an event campaign 'foo'
```

Comment:
Style nitpick: The comments above use title case for ActionNetwork entities, and also capitalize the first word; let's be consistent